<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>1037</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9" stretch="10,2">
         <item>
          <layout class="QVBoxLayout" name="graph_layout">
           <property name="spacing">
            <number>-1</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QTabWidget" name="tp_algorithm">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="tab_interpol">
              <attribute name="title">
               <string>Интерполяция</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
               <property name="bottomMargin">
                <number>11</number>
               </property>
               <item>
                <widget class="QGroupBox" name="groupBox_5">
                 <property name="title">
                  <string/>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QGroupBox" name="groupBox_4">
                    <property name="title">
                     <string>Количество точек</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
                     <property name="topMargin">
                      <number>9</number>
                     </property>
                     <item>
                      <widget class="QSpinBox" name="sb_dot_amount">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="focusPolicy">
                        <enum>Qt::ClickFocus</enum>
                       </property>
                       <property name="maximum">
                        <number>99999</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pb_interpolate">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Интерполировать</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">pb_drawing_group</string>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_approx">
              <attribute name="title">
               <string>Аппроксимация</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
               <property name="bottomMargin">
                <number>11</number>
               </property>
               <item>
                <widget class="QGroupBox" name="groupBox_11">
                 <property name="title">
                  <string/>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_10">
                  <property name="spacing">
                   <number>-1</number>
                  </property>
                  <property name="leftMargin">
                   <number>12</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QGroupBox" name="groupBox_6">
                      <property name="title">
                       <string>Количество точек</string>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_2">
                       <item>
                        <widget class="QSpinBox" name="sb_approx_dot_amount">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="maximum">
                          <number>999999</number>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QGroupBox" name="groupBox_7">
                      <property name="title">
                       <string>Степень полинома</string>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_4">
                       <item>
                        <widget class="QSpinBox" name="sb_polynom_degree">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QGroupBox" name="groupBox_8">
                      <property name="title">
                       <string>Количество дней продления</string>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_5">
                       <item>
                        <widget class="QSpinBox" name="sb_extension">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="maximum">
                          <number>9999</number>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pb_approximate">
                    <property name="text">
                     <string>Аппроксимировать</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">pb_drawing_group</string>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>Вычисление ожидаемого значения по дате</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QDateTimeEdit" name="date_picker">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::ClickFocus</enum>
                   </property>
                   <property name="showGroupSeparator" stdset="0">
                    <bool>false</bool>
                   </property>
                   <property name="dateTime">
                    <datetime>
                     <hour>0</hour>
                     <minute>0</minute>
                     <second>0</second>
                     <year>2000</year>
                     <month>2</month>
                     <day>1</day>
                    </datetime>
                   </property>
                   <property name="currentSection">
                    <enum>QDateTimeEdit::YearSection</enum>
                   </property>
                   <property name="displayFormat">
                    <string>yyyy/MM/dd h:mm</string>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pb_calc_by_date">
                   <property name="text">
                    <string>Вычислить</string>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">pb_drawing_group</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>Результат:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_result_calc_by_date">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QPushButton" name="pb_load">
                   <property name="text">
                    <string>Загрузить</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pb_clear">
                   <property name="text">
                    <string>Очистить</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="pb_drawing_group"/>
 </buttongroups>
</ui>
